app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,273 @@
+    <string name="description_email">De gescande QR-code bevat een e-mailadres.</string>
+    <string name="description_send_email">De gescande QR-code bevat een ingevuld e-mailadres.</string>
+    <string name="description_product">De gescande QR-code bevat een productcode.</string>
+    <string name="description_tel">De gescande QR-code bevat een telefoonnummer.</string>
+    <string name="description_sms">De gescande QR-code bevat een sms.</string>
+
+    <string name="description_text">De gescande QR-code bevat platte tekst.</string>
+
+    <string name="choose_action">Doorgaan</string>
+    <string name="abort_action">Afbreken</string>
+
+    <string name="proceed">Doorgaan</string>
+    <string name="back">Terug</string>
+
+    <string name="choose_action_url">Doorgaan</string>
+    <string name="abort_action_url">Afbreken</string>
+
+    <string-array name="text_array">
+        <item>Opzoeken op internet</item>
+        <item>Niets doen</item>
+    </string-array>
+    <string-array name="email_array">
+        <item>E-mail versturen</item>
+        <item>Toevoegen aan contactpersonen</item>
+    </string-array>
+    <string-array name="vcard_array">
+        <item>Toevoegen aan contactpersonen</item>
+        <item>Niets doen</item>
+    </string-array>
+    <string-array name="send_email_array">
+        <item>E-mail versturen</item>
+        <item>Toevoegen aan contactpersonen</item>
+    </string-array>
+    <string-array name="sms_array">
+        <item>SMS versturen</item>
+        <item>Nummer bellen</item>
+        <item>Toevoegen aan contactpersonen</item>
+    </string-array>
+    <string-array name="mms_array">
+        <item>MMS versturen</item>
+        <item>Nummer bellen</item>
+        <item>Toevoegen aan contactpersonen</item>
+    </string-array>
+    <string-array name="tel_array">
+        <item>Nummer bellen</item>
+        <item>Toevoegen aan contactpersonen</item>
+    </string-array>
+    <string-array name="geo_array">
+        <item>Maps-app openen</item>
+        <item>Niets doen</item>
+    </string-array>
+    <string-array name="url_array">
+        <item>URL openen</item>
+        <item>Niets doen</item>
+    </string-array>
+    <string-array name="product_array">
+        <item>Opzoeken op internet</item>
+        <item>Niets doen</item>
+    </string-array>
+    <string-array name="wifi_array">
+        <item>Wachtwoord kopi?ren</item>
+        <item>Niets doen</item>
+    </string-array>
+    <string-array name="history_array">
+        <item>Kopi?ren naar klembord</item>
+        <item>Item verwijderen</item>
+    </string-array>
+    <string-array name="generator_array">
+        <item>E-mailadres</item>
+        <item>URL</item>
+        <item>Telefoonnummer</item>
+        <item>SMS</item>
+        <item>Geolocatie</item>
+        <item>MeCard</item>
+        <item>BizCard</item>
+        <item>MMS</item>
+        <item>Wi-Fi</item>
+        <item>VCard</item>
+        <item>Winkel</item>
```

Comment:
This is meant for app-URLs, like com.secuso.privacyFriendlyCodeScanner.
To be opened by PlayStore/F-Droid Market Apps. 
Is Winkel a good description of it, so users will realize that?
(Original "Market" might not be the best description either)

Reply:
I thought it was about something like a supermarket, but I changed it to "App-winkel" ("App store") now. Thanks for the clarification! :)